Source/WebInspectorUI/UserInterface/Views/DOMTreeElement.js

```diff
@@ -1271,18 +1271,37 @@ WI.DOMTreeElement = class DOMTreeElement extends WI.TreeElement
             return;
         }
 
-        var treeOutline = this.treeOutline;
-        var wasExpanded = this.expanded;
+        let treeOutline = this.treeOutline;
+        let wasExpanded = this.expanded;
+        let [newTagName, ...attributes] = newText.trim().split(" ");
```

Comment:
what if there's multiple spaces in a row?

Reply:
That's what the `map` is for below, to remove additional space around potential attributes:

```
attributes.map(attribute => attribute.trim()).join(" ")
```

Reply:
looking more closely at `InspectorDOMAgent::setAttributesAsText` i dont think we actually even need to do this because it'll get handled for us in the backend

as such, we probably would be better off either only separating after the first whitespace
```js
let newTagName = newText;
let attributes = "";
let whitespaceIndex = newTagName.search(/\s/);
if (whitespaceIndex !== -1) {
    newTagName = newTagName.slice(0, whitespaceIndex);
    attributes = newTagname.slice(whitespaceIndex + 1).trim();
}

...


const name = "";
node.setAttribute(name, attributes, attributesAddedCallback);
```

btw we dont need to `newText.trim()` because that happens right before this